dasharo-security/measured-boot.robot

```diff
@@ -18,25 +18,376 @@ Resource            ../keys.robot
 # - document which setup/teardown keywords to use and what are they doing
 # - go threough them and make sure they are doing what the name suggest (not
 # exactly the case right now)
-Suite Setup         Run Keyword
-...                     Prepare Test Suite
+Suite Setup         Measured Boot Suite Setup
 Suite Teardown      Run Keyword
 ...                     Log Out And Close Connection
 
 
+*** Variables ***
+${PCRS_TO_CHECK}=       [0-9]|14
+
+
 *** Test Cases ***
 MBO001.001 Measured Boot support (Ubuntu 20.04)
     [Documentation]    Check whether Measured Boot is functional and
-    ...    measurements are stored into the TPM.a.
-    Skip If    not ${MEASURED_BOOT_SUPPORT}    MBO001.001 not supported
-    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    MBO001.001 not supported
+    ...    measurements are stored into the TPM.
+    Execute Command In Terminal    shopt -s extglob
+    ${hashes}=    Execute Command In Terminal
+    ...    grep . /sys/class/tpm/tpm0/pcr-sha@(1|256)/[0-3]
+    Should Not Contain Any    ${hashes}
+    ...    0000000000000000000000000000000000000000
+    ...    FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+    ...    No such file or directory
+    ...    error
+    ...    ignore_case=${TRUE}
+
+MBO002.001 Check if event log PCRs match actual values (Ubuntu 22.04)
```

Comment:
Not sure that `(Ubuntu 22.04)` here and above are necessary if it's the only OS that's being checked and we don't have anything specific to this particular OS in the test case.

Reply:
Removed in [aee905e](https://github.com/Dasharo/open-source-firmware-validation/pull/326/commits/aee905e0b6173c965ad1301559190fcc657c8da8) as currently whole test suite is run in Ubuntu